doc/integrator/ngeo.rst.tmpl

```diff
@@ -124,7 +124,14 @@ The sub section is the interface name, and after that we have:
 
 * ``lang_urls_suffix`` suffix used in l10n URL, see: :ref:`integrator_multi_organization`.
 
-The dynamic values names are: ``interface``, ``cache_version``, ``lang_urls``, ``fulltextsearch_groups``.
+The dynamic values names are:
+
+* ``interface``: The interface name given in parameters.
```

Comment:
I don't understand the "given in parameters"

Reply:
=> 'given in parameters of the dynamic view' ...

Reply:
Is it ok @jwkaltz ?

Reply:
Ah, yes. But "given in the parameters" instead of "given in parameters"

Reply:
Thanks :-)